File: ex01/java/main.java
public class main {
    public static void main(String[] args) {
        int sum = 0;
        for (int i = 0; i < 1000; i++) {
            if ((i % 3 == 0) || (i % 5 == 0)) {
                sum += i;
            }
        }
        System.out.printf("Sum of all divisors of 3 and 5 below 1000: %d\n", sum);

        // Time of O(n)
    }
}

File: ex01/rust/src/main.rs
fn main() {
    let mut sum = 0;
    for mut i in 1..1000 {
        if (i % 3 == 0) || (i % 5 == 0) {
            sum += i;
        }
    }
    println!("The sum of the divisors of 3 and 5 below 1000 is: {}", sum);
}